### DISCUSSION TITLE
Hannspree HF225 and AOC 2217, Twinkle Tray vs Monitorian

### DISCUSSION THREAD

**Discussion by cbmira01** [xanderfrangos/twinkle-tray#752] [Monitor Compatibility Issues]
I am using Windows 10, and I have two monitors: a Hannspree HF225 and an AOC 2217.
Both of the monitors have the DDC/CI feature, and both are enabled. 
Both monitors hang on one AMD Radeon 5450 graphics card via DVI-I to 2x VGA.

I tried Twinkle Tray versions 1.15.4 and 1.16.-beta5.3.
Both versions detect both monitors (DDC/CI green checkbox shows checked)
but brightness and contrast sliders for these monitors are not presented.

I have tried Twinkle Tray's various debugging steps and alternate settings,
with no luck.

Monitorian 4.6.0.0 does present brightness and contrast sliders for both monitors, 
and allows brightness and contrast to be adjusted for both monitors.

THANKS

**Comment by xanderfrangos** [comment]
In v1.16.0-beta5.3, could you generate a report from the Debug menu?:

![Screenshot 2024-01-07 174026](https://github.com/xanderfrangos/twinkle-tray/assets/33106561/22de6647-091c-427a-b6b3-e760e2f09d43)

Thanks!
